docs/eventlisteners.md

```diff
@@ -257,12 +256,16 @@ spec:
     - name: cel-trig
       interceptors:
         - cel:
-            filter: "header.match('X-GitHub-Event', 'push')"
+            filter: "headers.match('X-GitHub-Event', 'pull_request')"
+            overlays:
+            - key: extensions.truncated_sha
```

Comment:
Would be nice to have an example where the expression returns something other than a string e.g. a JSON object itself

Reply:
From a subtree of the evaluated body?

Reply:
well, Billy and I were playing around with this yesterday and we thought we needed to append a JSON body but we got around with just a string in the end. This is more of a feature so maybe we just do strings for now and hold of on anything more complex until we have a good use case for it!